modules/installation-aws-upi-rhcos-ami.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-rhcos-ami-{context}"]
+= {op-system} AMIs for the AWS UPI infrastructure
+
+You must use a valid {op-system-first} AMI for your Amazon Web Services
+(AWS) zone for your {product-title} nodes.
+
+.{op-system} AMIs
```

Comment:
#forum-coreos says that these AMIs are out of date and to use https://github.com/openshift/installer/pull/1612/files